cmd/limactl/gendoc.go

```diff
@@ -38,11 +41,33 @@ func genmanAction(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	outputType, err := cmd.Flags().GetString("type")
+	if err != nil {
+		return err
+	}
 	homeDir, err := os.UserHomeDir()
 	if err != nil {
 		return err
 	}
 	dir := args[0]
+	switch outputType {
+	case "man":
+		if err := genMan(cmd, dir); err != nil {
+			return err
+		}
+	case "docsy":
```

Comment:
I think this should be "markdown", and genMarkdown

Reply:
The generated contents are md. But in the generated md we customise it for using it with docsy. 

That's why i kept it as docsy. So that later if we need to render markdown as is it will be fine